app/src/main/java/com/rage/clamber/Fragments/ClimbsFragment.java

```diff
@@ -130,4 +133,112 @@ public void onFailure(Call<List<Climb>> call, Throwable t) {
         }
     }
 
+    /**
+     * Method called from the ClimbRecyclerViewAdapter when the Project Checkbox is clicked. It returns the
+     * climb Id and boolean value for whether or not the box is checked. With that information, a network
+     * call is made to either POST a new project to the database or DELETE an existing project for the user.
+     * @param climbId - the id of the climb where the project checkbox was selected
+     * @param isChecked - the boolean value of the project checkbox.
+     */
+    @Override
+    public void onProjectCheckBoxClicked(int climbId, boolean isChecked) {
+        UserClimbDataRequest request = new UserClimbDataRequest();
+        request.setClimbId(climbId);
+        request.setUsername(mainUser.getUserName());
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl(HomePage.CONNECTION_WEB_ADDRESS)
+                .addConverterFactory(JacksonConverterFactory.create())
+                .build();
+        ClamberService clamber = retrofit.create(ClamberService.class);
+        Log.d(TAG, "The id is: " + climbId + " and isChecked is: " + isChecked);
+        if (!isChecked) {
+
+            final Call<Project> createProjectCall = clamber.createProject(request);
+            createProjectCall.enqueue(new Callback<Project>() {
+                @Override
+                public void onResponse(Call<Project> call, Response<Project> response) {
+                    Log.d(TAG, "Successfully set up project for user: " + mainUser.getUserName());
+                }
+
+                @Override
+                public void onFailure(Call<Project> call, Throwable t) {
+                    Log.d(TAG, "Failed to set up project for user:" + mainUser.getUserName());
```

Comment:
include `t` to the `Log.d` call to ensure you have the stack trace if you ever hit this situation